Cubical/Categories/Instances/PosetCat/Base.agda

```diff
@@ -0,0 +1,45 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Instances.PosetCat.Base where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Categories.Category
+open import Cubical.Relation.Binary.Poset
+
+open import Cubical.Categories.Instances.PosetCat.Monotone
+open import Cubical.Categories.Instances.PosetCat.MonotoneAdjoint
+
+private
+  variable
+    ℓ ℓ' : Level
+
+open Category
+
+-- Category of Posets
+PosetCat : {ℓ ℓ' : Level} → Category
+  (ℓ-max (ℓ-suc ℓ) (ℓ-suc ℓ')) (ℓ-max ℓ ℓ')
+PosetCat {ℓ} {ℓ'} = record
+  { ob = Poset ℓ ℓ'
+  ; Hom[_,_] = λ X Y → MonFun X Y
+  ; id = MonId
+  ; _⋆_ = MonComp
+  ; ⋆IdL = λ {X} {Y} f → eqMon f f refl
+  ; ⋆IdR = λ {X} {Y} f → eqMon f f refl 
+  ; ⋆Assoc = λ {X} {Y} {Z} {W} f g h → eqMon _ _ refl
+  ; isSetHom = MonFunIsSet
+  }
+
+
+-- Category of Posets w/ Adjoints
+PosetCatAdjCycl : {ℓ ℓ' : Level} → Category
```

Comment:
Probably worth defining a general subcategory construction that abstracts this at some point. Then it would be easier to define all 3 of monotone functions with left adj, right adj or both

Reply:
Universe levels should be explicit again

Reply:
Made the universe explicit, but think the subcategory construction is a more significant refactor that I might save for another PR. Looks like we'll need to rework it anyways with the new internal Heyting algebra stuff.

Reply:
Opened #32 to track this refactor.